Lib/fontTools/feaLib/parser_test.py

```diff
@@ -1263,22 +1260,9 @@ def test_empty_statement_ignored(self):
         doc = self.parse(";;;")
         self.assertFalse(doc.statements)
 
-    def setUp(self):
-        self.tempdir = None
-        self.num_tempfiles = 0
-
-    def tearDown(self):
-        if self.tempdir:
-            shutil.rmtree(self.tempdir)
-
     def parse(self, text):
-        if not self.tempdir:
-            self.tempdir = tempfile.mkdtemp()
-        self.num_tempfiles += 1
-        path = os.path.join(self.tempdir, "tmp%d.fea" % self.num_tempfiles)
-        with open(path, "w", encoding="utf-8") as outfile:
-            outfile.write(text)
-        return Parser(path).parse()
+        featurefile = UnicodeIO(tounicode(text))
```

Comment:
We are importing unicode_literals from __future__, so I think tounicode() should not be needed here.